lib/spdy-transport/connection.js

```diff
@@ -273,9 +290,15 @@ Connection.prototype._handleFrame = function _handleFrame(frame) {
     // Load created one
     stream = state.stream.map[frame.id];
 
-    // RST or PRIORITY maybe out of sync
-    if (!stream && (frame.type === 'RST' || frame.type === 'PRIORITY'))
+    if (stream === null) {
```

Comment:
Nope, there is a case in h2spec that must be handled different with "non-exist" and "closed" streams. Can you propose another solution?